VimeoNetworking/Sources/NSError+Extensions.swift

```diff
@@ -192,20 +191,20 @@ public extension NSError
         /// Returns the api error code of the failing response, if available
     public var vimeoServerErrorCode: Int?
     {
-        if let errorCode = (self.userInfo[self.dynamicType.VimeoErrorCodeKeyLegacy] as? NSNumber)?.integerValue
+        if let errorCode = (self.userInfo[type(of: self).VimeoErrorCodeKeyLegacy] as? NSNumber)?.intValue
```

Comment:
Since I need to return the value an int anyway, I can just do:

`VimeoErrorCodeKeyLegacy] as? Int`